### PR TITLE
Feature/nav styling

### PR DESCRIPTION
**Pull request by katieolson84**
## Description

I updated styling to the nav side-bar. I was able to get the custom icons to appear in the nav by placing them as a background image. I am still working on getting the icons to change color when active (a new svg that is blue is needed). There are a few minor tweaks that still need to be made (submenu when selected and clicked, turns light blue). I also updated the svgs so the fill color is #cacccf and removed a few duplicates. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Login/_LoginContainerStyled.less [hunk 1]
```diff
@@ -31,7 +31,6 @@
   }
 
   .login-page {
-    // width: 50%;
     display: flex;
```
Comment (paulstgermain) [inline src/components/pages/Login/_LoginContainerStyled.less:34]: Always be cleaning the code - nice catch removing the noted out code here!

File: src/components/pages/Login/_SignupPageStyled.less [hunk 2]
```diff
@@ -72,8 +73,8 @@ button.buttonStyles {
   padding: 1.5rem;
   &:hover {
     background-color: #ffffff;
-    color: #bd5a27;
-    border: 1px solid #bd5a27;
+    color: @button;
+    border: 1px solid @button;
```
Comment (paulstgermain) [inline src/components/pages/Login/_SignupPageStyled.less:77]: Love the reuse of LESS variables, something I love to see and need to remember to do more personally!

**Review by paulstgermain — APPROVED** [review]
Nice progress here iterating on the sidebar implementation - all code added is simple and clean, and the extra effort to clean up around the added code just sweetens the deal! I see no problems here!

File: src/components/pages/Login/_SignupPageStyled.less (hunk 2 above)
Comment (MichelleCCodes) [reply src/components/pages/Login/_SignupPageStyled.less:77]: Agreed! Nice reuse, and clean up of the styling code here!

File: src/components/pages/SideDrawer/SideDrawer.js [hunk 3]
```diff
@@ -33,42 +22,58 @@ function SideDrawer() {
...
-        {React.createElement(collapsed ? MenuUnfoldOutlined : MenuFoldOutlined)}
-      </Button>
-
+    <div style={{ width: 246 }}>
       <Menu
         defaultSelectedKeys={['none']}
-        defaultOpenKeys={['saved', 'admin-tools']}
         mode="inline"
-        theme="dark"
+        theme="light"
```
Comment (MichelleCCodes) [inline src/components/pages/SideDrawer/SideDrawer.js:29]: Great job refactoring these components and changing the theme to better match overall styling!

**Review by MichelleCCodes — APPROVED** [review]
Nice work! Love the implementation of global styling. I'll have to be sure to ask you for guidance on using global styles when I work on new components!

**Pull request merged**